### PR TITLE
docs: add atk's official solid test case in examples

### PR DESCRIPTION
**Pull request by atk**
The last changes left my branch too detached, so I started anew and now this can serve as an official example. Thank you for your help and patience.

### CODE REVIEW

**Comment by atk** [comment]
Sorry, forgot to overwrite a local dependency that is already released. Give me a few minutes.

File: examples/solid/test/Hello.test.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  test('renders', () => {
+    const { container, unmount } = render(() => <Hello count={4} />)
+    expect(container.innerHTML).toMatchSnapshot()
+    unmount()
+  })
+
+  test('updates', async() => {
+    const { container, unmount, queryByText } = render(() => <Hello count={4} />)
+    const button = queryByText('x1')
+    const buttonClicked = new Promise((resolve) => {
```
Comment (sheremet-va) [inline examples/solid/test/Hello.test.jsx:15]: you forgot to await it
i would suggest using expect.anyAsserions() to check if expect is actually called

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: examples/solid/test/Hello.test.jsx (hunk 1 above)
Comment (atk) [reply examples/solid/test/Hello.test.jsx:15]: I do await it on line 23. This is intentional.

**Review by sheremet-va — APPROVED** [review]

**Comment by patak-dev** [comment]
Why is there a package-lock and ui types change in this PR? Should we separate them if they are unrelated?

**Comment by atk** [comment]
I thought I removed package.lock (accidentially used npm instead of pnpm once). Let me remove it.

**Pull request merged**